inventory

```diff
@@ -0,0 +1,17 @@
+[all]
+pr1
+pr2
+pr3
+
+[all:vars]
+ansible_user="root"
+
```

Comment:
[keepalived_nodes]
pr1
pr2
pr3

Additional groups for applying docker_registry server role. We don't need docker registry to be deployed on all other nodes except of those, who take part in keepalived cluster